centos_ci.sh

```diff
@@ -28,18 +28,10 @@ yum -y install \
   python-setuptools
 
 
-# Install Docker (https://docs.docker.com/engine/installation/linux/centos/#install-with-yum)
-sudo tee /etc/yum.repos.d/docker.repo <<-'EOF'
-[dockerrepo]
-name=Docker Repository
-baseurl=https://yum.dockerproject.org/repo/main/centos/7/
-enabled=1
-gpgcheck=1
-gpgkey=https://yum.dockerproject.org/gpg
-EOF
-
-yum -y install docker-engine
-systemctl enable docker.service
+# Install Docker (https://docs.docker.com/engine/installation/linux/centos/#install-using-the-repository)
+yum-config-manager --add-repo https://download.docker.com/linux/centos/docker-ce.repo
```

Comment:
-1 , we should never you upstream docker repo to install docker, that's plan wrong. For our ISO and even for testing our features we should always use what distro have in the repo.